packages/adminrouter/extra/src/.flake8

```diff
@@ -0,0 +1,13 @@
+[flake8]
+
+# Ignore the following errors:
```

Comment:
Discussed in person:
I will talk with @mellenburg  about unifying it. The idea is to have DC/OS create general set of rules to which all the packages need to adhere to. Packages themselves may be stricter if they want to, but never move lenient. The flake8 run will take place both in code-quality and Jenkins tests.